core/arch/arm/include/kernel/delay_arch.h

```diff
@@ -30,23 +30,18 @@
 #ifndef __KERNEL_DELAY_ARCH_H
 #define __KERNEL_DELAY_ARCH_H
 
+#ifdef CFG_CORE_HAS_GENERIC_TIMER
 #include <arm.h>
-#include <stdbool.h>
 #include <stdint.h>
 
-static inline uint64_t arm_cnt_us2cnt(uint32_t us)
+static unsigned int delay_cnt_freq(void)
```

Comment:
Why not "static inline"?
Same question below.

Reply:
my mistake. thanks to point that out.